app/main/menus.js

```diff
@@ -203,14 +205,7 @@ function otherMenuFile () {
   }, {
     label: i18n.t('&About Appium'),
     click () {
```

Comment:
`click: showAppInfoClickAction,` should also work fine

Reply:
ah... I tried `click: showAppInfoClickAction()` but it called at launching time, but `click: showAppInfoClickAction` worked as expected. 🤯